glide.yaml

```diff
@@ -23,7 +23,7 @@ import:
 - package: github.com/revel/revel
 - package: github.com/robfig/config
 - package: github.com/youtube/vitess
-  version: f729316a75213a533287dfe100b4a81177629b15
+  version: 9b42ea02086d8061f2fd49df6a00670689485fc7
```

Comment:
Just drop the `version`, the only reason it's there was because I didn't had time to update code to new version of vitess.